synapse/models/orgs.py

```diff
@@ -162,6 +162,9 @@
                 'doc': 'A position within an org which can be organized into an org chart with replacable contacts.'}),
 
             ('ou:attendee', ('guid', {}), {
+                'interfaces': (
+                    ('entity:contactable', {'template': {'contactable': 'attendee'}}),
+                ),
```

Comment:
By inheriting entity:contactable, ou:attendee will inherit :lifespan in addition to its :period property. Not a blocker but a heads up that this already occurs in a few other cases (e.g., inet:service:tenant, inet:service:account) that need to be resolved. "Lifespan" as a property name makes "less sense" for these forms so there is prolly a more general inheritance discussion that can happen separately to twiddle these.